### PR TITLE
Support Truffleruby in CI

### PR DESCRIPTION
**Pull request by gogainda**

### CODE REVIEW

**Comment by gogainda** [comment]
@eregon I need your help here, not sure why but truffle ruby head version is not working for github actions

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ jobs:
   ruby:
     strategy:
       matrix:
-        ruby: [2.5.x, 2.6.x, 2.7.x]
+        ruby: [2.5.x, 2.6.x, 2.7.x, truffleruby-head]
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - uses: actions/setup-ruby@v1
```
Comment (eregon) [inline .github/workflows/ci.yml:11]: It needs to be `ruby/setup-ruby@v1`, i.e., the Ruby-official action.

**Comment by gogainda** [comment]
@eregon works now

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,11 +4,11 @@ jobs:
...
+        ruby: [2.5, 2.6, 2.7, truffleruby-head]
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-ruby@v1
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
     - run: |
```
Comment (eregon) [inline .github/workflows/ci.yml:14]: Minor: `gem install bundler` below is not needed with `ruby/setup-ruby`.

**Comment by gogainda** [comment]
@Jbern16 @davishmcclurg please review/merge

**Comment by davishmcclurg** [comment]
Thanks!

**Pull request merged**